test/test_suit/test_snowpipe_streaming_string_avro_sr.py

```diff
@@ -0,0 +1,91 @@
+from test_suit.test_utils import RetryableError, NonRetryableError
+from time import sleep
+from confluent_kafka import avro
+
+# SR -> Schema Registry
+# Runs only in confluent test suite environment
+class TestSnowpipeStreamingStringAvroSR:
```

Comment:
Might be worth to run with and without schema registry?

Reply:
> But since we got away with SnowflakeConverterWithoutSchemaRegistry, we are implicitly saying we wont support avro without SR.

hmmm, this sounds like an issue, why we plan to stop supporting this? Do we know how many customers are using AVRO without SR before making this decision?

Reply:
Because of error handling. This was already decided when we went ahead with a plan of deprecating this custom snowflake converters because it will not support error handling to DLQ. 

There are other open source converters like https://github.com/farmdawgnation/registryless-avro-converter

Which can be used. 

> Do we know how many customers are using AVRO without SR before making this decision?
Nope. May be we can ask this to confluent. 

Also, we will not support for Snowpipe Streaming only - which is only PrPr. For snowpipe it is still supported and hopefully we will get more metrics from the changes I did - but requires customers to update it to latest version.